docs/external-template-creation.md

```diff
@@ -140,44 +140,56 @@ Once you have the story JSON, several code changes are needed to add it to the l
-
-   ```js
-    //...
-    /**
-     * External dependencies
-     */
-    import { __, _x } from '@web-stories-wp/i18n';
-
-    /**
-     * Internal dependencies
-     */
-    /* eslint-disable-next-line import/no-unresolved */
-    import { default as template } from './template';
-
-    export default {
-      title: _x('Your Template Title', 'template name', 'web-stories'),
-      tags: [
-        _x('Tags', 'template keyword', 'web-stories'),
-      ],
-      // Array of color objects with name and hex values.
-      colors: [
-        { label: _x('Blue', 'color', 'web-stories'), color: '#1f2a2e' },
-      ],
-      description: __(
-        'A short text describing your story template.',
-       'web-stories'
-      ),
-      ...template,
-      vertical: _x('Vertical name', 'template vertical', 'web-stories'),
-    };
-   ```
-
-3. In [`packages/templates/src/getTemplates.js`](https://github.com/google/web-stories-wp/blob/main/packages/templates/src/getTemplates.js), add `"<template_name>"` to the string array in the `getTemplates()` function.
-4. Verify in your WP environment that the new template is visible in the editor's "Explore Templates" section.
-5. Create a single pull request with all of the changes in steps 1-3.
+      
+2. Create a new file `metaData.js` in your newly created `<template_name>` directory. Your `<template_name>/metaData.js` file would then look something like this with object corresponding to the new template and properties `id`, `title`, `tags`, `colors`, `creationDate`, etc. 
+
+    ```javascript
+       //...
+      /**
+       * External dependencies
+       */
+      import { __, _x } from '@web-stories-wp/i18n';
+
+      export default {
+        slug: 'template-name',
+        creationDate: '2021-07-12T00:00:00.000Z',
+        title: _x('Your Template Title', 'template name', 'web-stories'),
+        tags: [
+          _x('Tags', 'template keyword', 'web-stories'),
+        ],
+        // Array of color objects with name and hex values.
+        colors: [
+          { label: _x('Blue', 'color', 'web-stories'), color: '#1f2a2e' },
+        ],
+        description: __(
+          'A short text describing your story template.',
+        'web-stories'
+        ),
+        vertical: _x('Vertical name', 'template vertical', 'web-stories'),
+      };
+    ```
+
+3. Create a new file `index.js` in your newly created `<template_name>` directory and import the `template.json` file and `metaData.js` file. Your `<template_name>/index.js` file would then look something like this:
+
+    ```javascript
+      //...
+      /**
+       * Internal dependencies
+       */
+      /* eslint-disable-next-line import/no-unresolved */
+      import { default as template } from './template';
+      /* eslint-disable-next-line import/no-unresolved */
+      import { default as metaData } from './metaData';
```

Comment:
FYI:

If you add `"import/no-unresolved": "off",` here:

https://github.com/google/web-stories-wp/blob/32725af74b887a4af122048bfba3502f8ecbcb6d/.eslintrc#L448-L465

Then we don't need those `eslint-disable-next-line` comments here. Makes it a bit more readable and easier to copy & paste.